d3a_interface/sim_results/area_throughput_stats.py

```diff
@@ -87,3 +90,26 @@ def update_results(self, area_dict, core_stats, current_market_time_slot_str):
         for child in area_dict['children']:
             if child['type'] == "Area":
                 self.update_results(child, core_stats, current_market_time_slot_str)
+
+    @staticmethod
+    def merge_results_to_global(market_trade: Dict, global_trade: Dict, *_):
```

Comment:
AFAIU, `*_` is a way to ignore all additional arguments, right?
Why is this needed here?

Reply:
Because `slot_list` from the abc is required only in some implementations of the `merge_results_to_global` method. Therefore if I created a method with only 2 arguments, Python would complain that the abstractmethod does not adhere to the same number of arguments as the base class method.